clientapi/routing/routing.go

```diff
@@ -94,6 +94,7 @@ func Setup(
 	unstableFeatures := map[string]bool{
 		"org.matrix.e2e_cross_signing": true,
 		"org.matrix.msc2285.stable":    true,
+		"org.matrix.msc3916":           true,
```

Comment:
Wondering if this should be `.stable`?

Reply:
Probably yes since the unstable endpoints aren't supported. I think clients will only use new endpoints if v1.11 is advertised though

Reply:
From a short test with Element Web: Even with v1.11 advertised, the new endpoints aren't used. :/

Reply:
Element Web specifically is buggy: element-hq/element-web#27759

Reply:
Works fine with ED nightly when advertising v1.11.